### DISCUSSION TITLE
Very cool

### DISCUSSION THREAD

**Discussion by afcmike** [starsite/SwiftFM#4] [Q&A]
Brian, I've had a lot of fun experimenting with SwiftFM while learning to build a SwiftUI app. I may have found a bug and may just have a question or two. I'd enjoy discussing with you if you're open to it. In any case, thanks so much for what you've done here.

-Mike

**Comment by afcmike** [comment]
Just FYI, @starsite I found a recording of the presentation you gave in January at FMPUG. It had the clue I needed.

**Comment by starsite** [comment]
Hi Mike. Whatever notice GitHub sent me, I apparently missed it (nothing in email), may have been lost in a sea of iOS notifications.

Glad to hear you're finding SwiftFM useful and that my presentation offered some guidance on the issue you ran into. Let me know if there's something I can clarify in code and/or documentation. Be happy to talk about it. If you're on FMSlug we can chat there, or I have a Slack for Starsite Labs, I can invite you via the email listed on your @afcmike profile , or a different one if you like.
